lib/ace/ext/papeeria/highlighter.coffee

```diff
@@ -10,7 +10,8 @@ define((require, exports, module) ->
             session.removeMarker(session.$bracketMismatchHighlight)
             session.$bracketMatchHighlight = null
             session.$bracketMismatchHighlight = null
-            toggleSurroundingBracketsPopup(editor)
+            if toggleSurroundingBracketsPopup
+                toggleSurroundingBracketsPopup(editor)
```

Comment:
In CoffeeScript you can use existential operator:

`toggleSurroundingBracketsPopup?(editor)`

However, do you really need this condition? Even if this argument is optional in `init` method, you can replace it with a dummy implementation which does nothing. You may even require the argument to be compulsory (e.g. throw error if it is undefined)